dtos/tags.go

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (C) 2023 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+import "encoding/xml"
+
+// Tags is the alias of map[string]any type
+type Tags map[string]any
+
+// MarshalXML fulfills the Marshaler interface for map[string]any type
+// which is not originally supported by xml.Marshal function
```

Comment:
Add `why` this is needed as was in the comment in the code that was removed from Event DTO